commands/video/version.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const serviceMetadata = JSON.parse(fs.readFileSync(`${__dirname}/../../package.json`));
+const subcommand = 'version';
+const category = 'video';
+
+module.exports = {
+  name: subcommand,
+  run: async (context) => {
+    await context.print.info('amplify-category-video@' + serviceMetadata.version);
```

Comment:
`await` is not needed for the `context.print.info`

`await` is meant for async calls but since the print doesn't need to be async it can be called sync.

Reply:
ah - gotcha, makes sense!